conda/common/path.py

```diff
@@ -234,8 +235,12 @@ def get_major_minor_version(string, with_dot=True):
     return ".".join(maj_min) if with_dot else "".join(maj_min)
 
 
+BIN_DIRECTORY = "Scripts" if on_win else "bin"
```

Comment:
That's where I considered to put it originally however per the docstrings:

https://github.com/conda/conda/blob/156a69dee64ce94f39c77271d3e558457e04fb43/conda/common/__init__.py#L4-L10

https://github.com/conda/conda/blob/156a69dee64ce94f39c77271d3e558457e04fb43/conda/base/__init__.py#L4-L19

It should be here in `conda.common.path` since this is not a conda specific parameter.